src/bundles/SellerRegistration/components/BusinessDetailsForm/BusinessDetailsForm.js

```diff
@@ -89,7 +88,7 @@ class BusinessDetailsForm extends BaseForm {
                               (<span>You need an Australian Business Number to do business in Australia.&nbsp;
                               <a href='https://abr.gov.au/For-Business,-Super-funds---Charities/Applying-for-an-ABN/Apply-for-an-ABN/' target="_blank" rel="external">Apply for an ABN here.</a>
                           </span>)}
-                          disabled="disabled"
+                          readOnly={!!supplierCode}
```

Comment:
yes, webpack told me to use it - !  converts the object being not null/null to true/false and then another negates that ie. readOnly is true if supplierCode is not null